docs/fsharp/style-guide/component-design-guidelines.md

```diff
@@ -0,0 +1,884 @@
+---
+title: F# component design guidelines
+description: Learn the guidelines for writing F# components intended for consumption by other callers.
+author: cartermp
```

Comment:
general comment:

@mairaw has been updating our global and folder metadata. The only tags you need are "title", "description", and "ms.date".  (You are the default author for the F# guide.

You can simplify all these files.